.github/workflows/e2e_ppo_trainer.yml

```diff
@@ -109,15 +109,15 @@ jobs:
       - name: Test merging FSDP checkpoints (Qwen Actor) 
         run: |
           exp_name="qwen2.5-0.5b-function-reward-minimal-fsdp8"
-          python scripts/model_merger.py test --backend fsdp --local_dir checkpoints/verl-test/${exp_name}/global_step_1/actor --test_hf_dir checkpoints/verl-test/${exp_name}/global_step_1/actor/huggingface
+          python -m scripts.model_merger test --backend fsdp --local_dir checkpoints/verl-test/${exp_name}/global_step_1/actor --test_hf_dir checkpoints/verl-test/${exp_name}/global_step_1/actor/huggingface
```

Comment:
this means that `scripts` become a pkg namespace. if you want to enable `python -m ` i would suggest using `python -m verl.xxx` instead.